proxy/src/main/java/org/apache/rocketmq/proxy/config/ProxyConfig.java

```diff
@@ -148,6 +148,7 @@ public class ProxyConfig implements ConfigFile {
     private int consumerProcessorThreadPoolNums = PROCESSOR_NUMBER;
     private int consumerProcessorThreadPoolQueueCapacity = 10000;
 
+    private boolean forceUseEndpointPort = false;
```

Comment:
how about `useEndpointPortFromRequest`?

Reply:
yes it should be more suitable，I'll modify it